ruddock/modules/rotation/routes.py

```diff
@@ -11,13 +11,19 @@
 @login_required(Permissions.ROTATION)
 def show_portal():
   dinner_list = helpers.DINNERS
+  buckets = helpers.BUCKETS
   return flask.render_template('portal.html',
-    dinner_list=dinner_list)
+    dinner_list=dinner_list, buckets=buckets)
 
-@blueprint.route('/dinner/<int:dinner_id>')
+@blueprint.route('/directory')
```

Comment:
Why not just have two separate endpoints? It's not like these really have any logic in common.